ani-cli

```diff
@@ -127,6 +127,19 @@ get_links () {
 	printf '%s' "$video_url" | sed -n -E "s/(.*)\.m3u8/\1.$video_quality.m3u8/p"
 }
 
+check_for_update () {
+	git fetch 
+	UPSTREAM=${1:-'@{u}'}
+	LOCAL=$(git rev-parse @)
+	REMOTE=$(git rev-parse "$UPSTREAM")
+
+	UPDATE_MESSAGE="UPDATE AVAILABLE\nPlease run the following commands in your ani-cli directory to update\n----------\ngit pull\nsudo make\n----------\n"
+
+	if [ $LOCAL != $REMOTE ]; then
+    echo $UPDATE_MESSAGE
```

Comment:
Can you fix the indentation here? Also, instead of `echo`, use `printf`.

Reply:
done and done